mlora.py

```diff
@@ -89,12 +89,23 @@ def setup_seed(seed):
 
 def load_base_model(config: Dict[str, any]) -> Tuple[mlora.Tokenizer, mlora.LLMModel]:
     if args.model_type == "llama":
-        model = mlora.LlamaModel.from_pretrained(
-            path=args.base_model,
-            device=args.device,
-            bits=(8 if args.load_8bit else (4 if args.load_4bit else None)),
-            log_fn=log
-        )
+        if "mixlora" in config and config["mixlora"]:
```

Comment:
`args.model_type == "mixloar"` is better, this will increase the cyclomatic complexity

Reply:
if there have many if branch, we can use the table driven method, like use cls.init(xxx).